common/dynamicconfig/constants.go

```diff
@@ -3821,6 +3842,11 @@ var BoolKeys = map[BoolKey]DynamicBool{
 		Description:  "EnableAuthorization is the key to enable authorization for a domain, only for extension binary:",
 		DefaultValue: false,
 	},
+	EnableTasklistIsolation: DynamicBool{
+		KeyName:      "system.enableIsolationGroupPartitioning",
```

Comment:
keyName is not updated.

Reply:
ah, that was sloppy of me, sorry, let me fix